app.js

```diff
@@ -76,12 +76,17 @@ function generateRandomImages(event) {
       button[i].removeEventListener('click', generateRandomImages);
     }
 
+    localStorage.removeItem('All Products');
+    localStorage.removeItem('Round');
+
     alert('Finished voting! View your results by clicking the results button.');
   }
   // Generate new random index 
   generateRandomIndex();
+
   // Local Storage
-  setAllProducts();
+  setItems();
+  getItems();
```

Comment:
If these two functions are in one place, you probably don't want them in this order.

Think of set like save, and get like load. Since both of these functions are trying to access the same thing, and we're saving before we're loading, we will always load the data we already have.

If we want to get data from a previous time we were on the page, we would want to getItems pretty close to when we start doing anything, and setItems whenever we want to update our save.